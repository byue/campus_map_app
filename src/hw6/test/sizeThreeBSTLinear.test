# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 3 linear graph.

CreateGraph characters
AddNode characters GORILLA-MAN
AddNode characters HAWK
AddNode characters VENUS-II

AddEdge characters GORILLA-MAN HAWK COMIC1
AddEdge characters HAWK VENUS-II COMIC2

FindPath characters GORILLA-MAN HAWK

FindPath characters HAWK VENUS-II

FindPath characters GORILLA-MAN VENUS-II

FindPath characters VENUS-II GORILLA-MAN

FindPath characters VENUS-II VENUS-II

FindPath characters VENUS-II GGGG

FindPath characters BBBB VENUS-II

FindPath characters BBBB GGGG