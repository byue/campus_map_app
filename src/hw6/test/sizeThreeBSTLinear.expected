# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 3 linear graph.

created graph characters
added node GORILLA-MAN to characters
added node HAWK to characters
added node VENUS-II to characters

added edge COMIC1 from GORILLA-MAN to HAWK in characters
added edge COMIC2 from HAWK to VENUS-II in characters

path from GORILLA-MAN to HAWK:
GORILLA-MAN to HAWK via COMIC1

path from HAWK to VENUS-II:
HAWK to VENUS-II via COMIC2

path from GORILLA-MAN to VENUS-II:
GORILLA-MAN to HAWK via COMIC1
HAWK to VENUS-II via COMIC2

path from VENUS-II to GORILLA-MAN:
no path found

path from VENUS-II to VENUS-II:

unknown character GGGG

unknown character BBBB

unknown character BBBB
unknown character GGGG